common/src/main/java/com/viaversion/viabackwards/protocol/v1_14to1_13_2/rewriter/BlockItemPacketRewriter1_14.java

```diff
@@ -467,10 +468,19 @@ public Item handleItemToClient(UserConnection connection, Item item) {
             if (lore != null) {
                 saveListTag(display, lore, "Lore");
 
-                for (StringTag loreEntry : lore) {
-                    String value = loreEntry.getValue();
-                    if (value != null && !value.isEmpty()) {
-                        loreEntry.setValue(ComponentUtil.jsonToLegacy(value));
+                final Iterator<StringTag> each = lore.iterator();
+                while (each.hasNext()) {
+                    try {
```

Comment:
Put the try catch around the loop to make the use more readable

Reply:
Done